ui/src/papers/index.tsx

```diff
@@ -162,17 +163,7 @@ const PaperList = () => {
             <TableBody>
               {papers
                 .filter((paper) => {
-                  return (
-                    paper.arxivId !== undefined &&
-                    [
-                      "1508.07252",
-                      "1706.03762",
-                      "1811.12160",
-                      "1010.2823",
-                      "1309.7440",
-                      "1404.6635",
-                    ].indexOf(paper.arxivId) !== -1
-                  );
+                  return paper.arxivId !== undefined;
```

Comment:
```suggestion
                  return !!paper.arxivId;
```
If we just want to make sure that the arxivId is truthy, this would be better than explicitly checking that it isn't `undefined`.

Reply:
Gotcha. I go either way on this one.  I tend to avoid value coercion where I can, because it can be tricky and cause unexpected side-effects.  In this case we know that it'll be `undefined` or a non-empty string, so I like the explicitness of checking for the case that I know we don't want.

That said by the same argument I could say that there's no risk in the value coercion -- so let's go with `!!` because it's shorter (and I imagine more in line with S2's JavaScript styles!).